README.md

```diff
@@ -1,10 +1,11 @@
-# DAOStack Alchemy - Creating DAOs for an Emergent Future
+# DAOstack Alchemy - Creating DAOs for a Collective Intelligence
 
 ## What is Alchemy?
 
-DAOStack Alchemy is a browser application that enables anyone with an account on the Ethereum blockchain, and STK tokens to spend, to easily create and work with DAOs on Ethereum that run on predefined, configurable smart contracts in a network of organizations, collaborators and investors that collaborate on the blockchain.
+DAOstack Alchemy is a browser application that enables anyone with an account on the Ethereum blockchain, and STK tokens to spend, to easily create and work with DAOs on Ethereum that run on predefined, configurable smart contracts in a network of organizations, collaborators and investors that collaborate on the blockchain.
 
 ## Implementation
+
 The application uses the standards-compliant and highly modular browser-side framework [Aurelia](http://aurelia.io) (hence "Aurelia" in the name of the repository).
 
 Browser-side the application uses TypeScript/ECMAScript 2016+.
```

Comment:
> n uses TypeScript/ECMAScript 2016+. 

Is that significant ?

Reply:
To coders, yes.

Reply:
I am not a coder..what is it ?

Reply:
Sorry, poor wording, I think you *are*  a coder, and a good one  :-)

What I meant was that it is important to website developers who might have an interest in coding the application, or who might just be curious what languages were used.  TypeScript is a  language, ECMAScript 2016 is a version of JavaScript.

However, I think to be safe, I think it should be 2015 (ES6), not 2016 (ES7), so I have changed it.